Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG ASYNCAPI_GENERATOR_VERSION=0.50.0
```

Comment:
why not `latest` ?

Reply:
It's possible that the latest version will cache and you will end up with a docker image with an invalid generator version. With an argument, you can test past versions locally and ensure that you install the right generator version. Seems like a declarative approach 😄 

PS. I also created a similar PR for CLI